internal/api/internal.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Google LLC
```

Comment:
@akremsa Please take a look at conversation about this file naming, which might be `statestore.proto` : https://github.com/googleforgames/open-match/pull/1279#discussion_r525408856